README.rst

```diff
@@ -217,11 +217,11 @@ Included in ``ansible-lint/examples`` are some example playbooks with undesirabl
 
     [ANSIBLE0002] Trailing whitespace
     examples/example.yml:13
-        action: do nothing   
+        action: do nothing
```

Comment:
@ssbarnea this change is wrong because it's a demo of trailing whitespace error.